src/api/ZoweExplorerApi.ts

```diff
@@ -42,7 +42,7 @@ export namespace ZoweExplorerApi {
          *      will use the profile the API was retrieved with by default
          * @returns {Session} a Zowe CLI Session
          */
-        getSession(profile?: IProfileLoaded): Session;
+        getSession(profile?: IProfileLoaded, baseProfile?: IProfile): Promise<Session>;
```

Comment:
I think @jellypuno mentioned this a while ago, but this change seems to particularly break profile validation and the enable/disable validation setting for extenders. Toggling the enable/disable validation setting, particularly the profile-specific one, seems to work sometimes but not others based on what shows in the context menu. Whichever way validation is set, extender profiles validate repeatedly, particularly for actions such as create or delete data set member, even if they already have a green checkmark by them or if validation appears "disabled" via the settings. 

That aside... The overall existing experience I tested for extenders seems to still work (i.e. create, edit, save, delete, favorite, etc.). Unit tests are all passing, but I am getting around 15 failing integration tests, mostly due to a combination of `500` errors and `ECONNREFUSED` errors. If others are not seeing the failing integration tests, it could just be on my end. I'm curious what others get for their integration tests. I tried running mine both with and without the SCS plugin installed, and got approximately the same result.

Perhaps this is something we could discuss at our next scrum.

Reply:
That sounds like a good idea. @zFernand0 for a reminder to address this in a parking lot (as I won't be there). Thanks @lauren-li!

Reply:
I think it sounds like profile validation is broken in some places on extenders only now, @lauren-li ?

I am also getting failed integration tests.